specification/cognitiveservices/data-plane/Luis/v2.0/LUIS Programmatic APIs v2.0.swagger.json

```diff
@@ -0,0 +1,5225 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "LUIS Programmatic APIs v2.0",
+    "version": "1.0"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{AzureRegion}.api.cognitive.microsoft.com",
+    "parameters": [
+      {
+        "name": "AzureRegion",
+        "description": "Supported Azure regions for Cognitive Services endpoints",
+        "x-ms-parameter-location": "client",
+        "required": true,
+        "type": "string",
+        "in": "path",
+        "x-ms-skip-url-encoding": true,
+        "x-nullable": "false",
+        "x-ms-enum": {
+          "name": "AzureRegions",
+          "modelAsString": false
+        },
+        "enum": [
+          "westus",
+          "westeurope"
+        ]
+      }
+    ]
+  },
+  "basePath": "/luis/api/v2.0",
+  "schemes": [
+    "https"
+  ],
+  "securityDefinitions": {
+    "apiKeyHeader": {
+      "type": "apiKey",
+      "name": "Ocp-Apim-Subscription-Key",
+      "in": "header"
+    },
+    "apiKeyQuery": {
+      "type": "apiKey",
+      "name": "subscription-key",
+      "in": "query"
+    }
+  },
+  "security": [
+    {
+      "apiKeyHeader": []
+    },
+    {
+      "apiKeyQuery": []
+    }
+  ],
+  "paths": {
+    "/apps/{appId}/versions/{versionId}/patterns": {
+      "post": {
+        "description": "Creates a new pattern feature.",
+        "operationId": "features - Create pattern feature",
+        "parameters": [
+          {
+            "name": "appId",
+            "in": "path",
+            "description": "Format - guid. The application ID.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "versionId",
+            "in": "path",
+            "description": "The version ID of the task.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "patternCreateObject",
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/PatternCreateObject"
+            },
+            "description": "A JSON object containing Name and Pattern of the feature."
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "responses": {
+          "201": {
+            "description": "The ID of the created feature.",
+            "examples": {
+              "application/json": "83147"
+            }
+          },
+          "400": {
```

Comment:
Are you intending to throw and exception in generated code? Please see https://github.com/Azure/azure-rest-api-specs/blob/2fb9a0b3b902335ff0b0033711c234431931ec9d/documentation/creating-swagger.md#default-response on how autorest acts on default response and negative responses